### PR TITLE
Review Web Share Target article.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/site/content/en/blog/web-share-target/index.md [hunk 1]
```diff
@@ -192,8 +192,8 @@ If the user selects your application, and your `method` is `"GET"` (the
 default), the browser opens a new window at the `action` URL. The browser then
 generates a query string using the URL-encoded values supplied in the manifest.
 For example, if the sharing app provides `title` and `text`, the query string is
-`?title=hello&text=world`. To process this, use a `DOMContentLoaded` event
-listener in your foreground page and parse the query string:
+`?title=hello&text=world`. To process this, use a [`DOMContentLoaded`](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)
```
Comment (rachelandrew) [inline src/site/content/en/blog/web-share-target/index.md:195]:
```suggestion
`?title=hello&text=world`. To process this, use a [`DOMContentLoaded`](https://developer.mozilla.org/docs/Web/API/Window/DOMContentLoaded_event)
```

File: src/site/content/en/blog/web-share-target/index.md [hunk 2]
```diff
@@ -219,7 +219,8 @@ provided in the manifest.
 
 The foreground page cannot process this data directly. Since the page sees the data as
 a request, the page passes it to the service worker, where you can intercept it with a
-`fetch` event listener. From here, you can pass the data back to the foreground
+[`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope#:~:text=Content%20Index.-,fetch,-Occurs%20when%20a)
```
Comment (rachelandrew) [inline src/site/content/en/blog/web-share-target/index.md:222]:
```suggestion
[`fetch`](https://developer.mozilla.org/docs/Web/API/ServiceWorkerGlobalScope#:~:text=Content%20Index.-,fetch,-Occurs%20when%20a)
```

**Review by rachelandrew — APPROVED** [review]
Just fixed a couple of MDN URLs.

**Comment by jpmedley** [comment]
PTAL

**Review by tomayac — APPROVED** [review]
LGTM, thanks, Joe!

**Pull request merged**